### PR TITLE
Add simple sprite animation example

### PR DESCRIPTION
**Pull request by kkristof**

### CODE REVIEW

File: examples/sprite/sprite.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+function sleep(milliseconds) {
+  var start = new Date().getTime();
+  for (var i = 0; i < 1e7; i++) {
+    if ((new Date().getTime() - start) > milliseconds){
```
Comment (szledan) [inline examples/sprite/sprite.js]: missing white-space: `...milliseconds) {...`

File: examples/sprite/sprite.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+spH = 132;  // sprite height
+sx = 0;
+sy = 0;
+sw = spW;
+sh = spH;
+dx = 0;
+dy = 318;
+dw = sw;
+dh = sh;
+for(i = 0; !surface.hasToQuit() && i < 21; i++) {
```
Comment (szledan) [inline examples/sprite/sprite.js]: missing space: `for (i = 0;...`

File: examples/sprite/sprite.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    sx = i % numOfSprites * spW;
+    gepard.fillRect(0, 0, surface.width, surface.height);
+    dx = i * bgStep % surface.width;
+    gepard.drawImage(background, 0, 0, background.width, background.height, surface.width - dx, 0, surface.width, surface.height);
+    gepard.drawImage(background, 0, 0, background.width, background.height, -dx, 0, surface.width, surface.height);
+    gepard.drawImage(image, sx, sy, sw, sh, 250, dy, dw, dh);
+    gepard.finish();
+    sleep(64);
+    if (!(i%5)){
```
Comment (szledan) [inline examples/sprite/sprite.js]: missing spaces: `...(!(i % 5)) {...`

**Review by szledan — CHANGES_REQUESTED** [review]

**Comment by kkristof** [comment]
@szledan Thanks for the review, I've fixed the style issues.

**Review by szledan — APPROVED** [review]
LGTM +1

**Review by bzsolt — APPROVED** [review]
LGTM

**Pull request merged**